ux.symfony.com/assets/styles/vendor/_highlight.scss

```diff
@@ -1,57 +1,74 @@
-.hljs-comment, .hljs-prompt {
-    color: #8b949e;
-    user-select: none;
-    -webkit-user-select: none
+/** tempest/highlight theme (https://github.com/tempestphp/highlight/blob/main/src/Themes/Css/dracula.css) */
```

Comment:
@smnandre, perhaps you can create a follow-up PR to adjust the styles/theme (I may have broken some stuff (ie `.codeblock-terminal .hljs-keyword`).